samples/sysbuild/hello_world/sample.yaml

```
@@ -5,8 +5,12 @@ sample:
     the board target they were run on.
 
 common:
-  build_only: true
   sysbuild: true
+  harness: console
+  harness_config:
+    type: multi_line
+    regex:
+      - "Hello world from"
```

```suggestion
      - "Hello world from .*"
```

wouldn't it be more accurate/clear?

---

Indeed, changed, thanks :)